### ISSUE TITLE
MultipleNegativesRankingLoss bug

### ISSUE THREAD

**Issue by dli1** [UKPLab/sentence-transformers#1689]
I found that in the `sentence_transformers.losses. MultipleNegativesRankingLoss `, there is probably a bug. The variable `labels` is not used at all. Is this intentional, or a bug?

See the code below:
```
class MultipleNegativesRankingLoss(nn.Module):
def forward(self, sentence_features: Iterable[Dict[str, Tensor]], labels: Tensor):
        reps = [self.model(sentence_feature)['sentence_embedding'] for sentence_feature in sentence_features]
        embeddings_a = reps[0]
        embeddings_b = torch.cat(reps[1:])
        scores = self.similarity_fct(embeddings_a, embeddings_b) * self.scale
        labels = torch.tensor(range(len(scores)), dtype=torch.long, device=scores.device)  # Example a[i] should match with b[i]
        return self.cross_entropy_loss(scores, labels)
```

**Comment by dli1** [comment]
There is no bug.

**Comment by PhilipMay** [comment]
So can it be closed? @dli1
